api/environments/views.py

```diff
@@ -209,11 +211,15 @@ class WebhookViewSet(
     mixins.UpdateModelMixin,
     mixins.DestroyModelMixin,
     viewsets.GenericViewSet,
+    TriggerSampleWebhookMixin,
 ):
     serializer_class = WebhookSerializer
     pagination_class = None
     permission_classes = [IsAuthenticated, NestedEnvironmentPermissions]
 
+    webhook_model = Webhook
+    sample_trigger_method = staticmethod(trigger_sample_environment_webhook)
```

Comment:
I don't really understand why we need to wrap this in a `staticmethod` here?

Reply:
Because in the mixin we call it like `   response = self.sample_trigger_method(webhook)` which results in self being the first argument passed to the function.

Reply:
Yeah, ok, I guess that makes sense. 

I think, however, what would make more sense is just using the WebhookType enum to determine which data to use. So there should only be a single `trigger_sample_webhook` method which takes the enum and determines the sample data to use from that. 

Similarly we could have a constant like `{<WebhookType>: <Model class>}` to determine the model instead of setting the model directly on the viewset.